rfcs/0667-apikey-delegated-permissions.md

```diff
@@ -0,0 +1,124 @@
+# API key delegated permissions
+
+Instead of maintaining separate principal types and access rules for regular users vs API keys, switch to the model whereby a user creates API keys and delegates one or more of the user's permissions to it.
```

Comment:
This reads a little clunky. What about:

_Unify access rules between regular users and API keys by switching to a model where API keys are delegated a subset of their owning user's permissions._

Reply:
Thanks, I'll make the change :+1: